diff --git a/index.d.ts b/index.d.ts
index af3ad561f4342a5ed3bcb1678bd289e33470b51f..56157f7710fd6f1c5141dedeece3492c17032787 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -38,6 +38,7 @@
 
 import * as CSS from 'csstype';
 import * as PropTypes from 'prop-types';
+// @ts-ignore
 import { Interaction as SchedulerInteraction } from 'scheduler/tracing';
 
 type NativeAnimationEvent = AnimationEvent;
diff --git a/ts5.0/index.d.ts b/ts5.0/index.d.ts
index 0ba0246adfa6beab20de06a06a25535e97687aa6..fcde381468c971e4ca51b8670df5df5a34141f22 100755
--- a/ts5.0/index.d.ts
+++ b/ts5.0/index.d.ts
@@ -6,6 +6,7 @@
 
 import * as CSS from 'csstype';
 import * as PropTypes from 'prop-types';
+// @ts-ignore
 import { Interaction as SchedulerInteraction } from 'scheduler/tracing';
 
 type NativeAnimationEvent = AnimationEvent;
