diff --git a/types/lodash/index.d.ts b/types/lodash/index.d.ts
index 1fa09484625c1062a9f23708b52dcbbbdc808ee2..510aca729e088309ed632deef629693fe00b7053 100644
--- a/types/lodash/index.d.ts
+++ b/types/lodash/index.d.ts
@@ -29,18 +29,17 @@ export as namespace _;
 
 declare const _: _.LoDashStatic;
 declare namespace _ {
-    // tslint:disable-next-line no-empty-interface (This will be augmented)
-    interface LoDashStatic {}
+  // tslint:disable-next-line no-empty-interface (This will be augmented)
+  interface LoDashStatic {}
 }
 
 // Backward compatibility with --target es5
 declare global {
-    // tslint:disable-next-line:no-empty-interface
-    interface Set<T> { }
-    // tslint:disable-next-line:no-empty-interface
-    interface Map<K, V> { }
-    // tslint:disable-next-line:no-empty-interface
-    interface WeakSet<T> { }
-    // tslint:disable-next-line:no-empty-interface
-    interface WeakMap<K extends object, V> { }
+  // tslint:disable-next-line:no-empty-interface
+  interface Set<T> {}
+  // tslint:disable-next-line:no-empty-interface
+  interface Map<K, V> {}
+  // tslint:disable-next-line:no-empty-interface
+  interface WeakSet<T> {}
+  // tslint:disable-next-line:no-empty-interface
 }
